metadata-models/src/main/pegasus/com/linkedin/chart/ChartQuery.pdl

```diff
@@ -0,0 +1,33 @@
+namespace com.linkedin.chart
+
+/**
+ * Information for chart query which is used for getting data of the chart
+ */
+record ChartQuery {
+
+  /**
+   * Raw query to build a chart from input datasets
+   */
+  rawQuery: string
+
+  /**
+   * Chart query type
+   */
+  type: enum ChartQueryType {
+
+    /**
+     * LookML queries
+     */
+    LOOKML
+
+    /**
+     * SQL type queries
+     */
+    SQL
+
+    /**
+     * Other query type
+     */
+    OTHER
```

Comment:
If the idea is to have an enum, I think we should encourage the expansion, rather than allow use of `OTHER`.